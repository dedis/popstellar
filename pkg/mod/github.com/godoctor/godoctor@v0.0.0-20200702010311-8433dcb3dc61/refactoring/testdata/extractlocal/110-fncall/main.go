package main

import "fmt"

func dbl(n int) int {
	return 2 * n
}

func main() {
	dbl(4) // <<<<< var,10,2,10,7,variable,pass
	fmt.Printf("Test\n")
}
