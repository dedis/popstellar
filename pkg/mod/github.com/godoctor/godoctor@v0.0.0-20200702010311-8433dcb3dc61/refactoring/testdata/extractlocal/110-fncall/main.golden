package main

import "fmt"

func dbl(n int) int {
	return 2 * n
}

func main() {
	variable := dbl(4)
	variable // <<<<< var,10,2,10,7,variable,pass
	fmt.Printf("Test\n")
}
